File: URI/Java/Animal.java
import java.util.Scanner;
import java.lang.Math;

public class Animal {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        String c1, c2, c3;
        c1 = input.nextLine();
        c2 = input.nextLine();
        c3 = input.nextLine();
        if (c1.compareTo("vertebrado") == 0) {
            if (c2.compareTo("ave") == 0) {
                if (c3.compareTo("carnivoro") == 0){
                    System.out.println("aguia");
                } else {
                    System.out.println("pomba");
                }
            } else {
                if (c3.compareTo("onivoro") == 0) {
                    System.out.println("homem");
                } else {
                    System.out.println("vaca");
                }
            }
        } else {
            if (c2.compareTo("inseto") == 0) {
                if (c3.compareTo("hematofago") == 0) {
                    System.out.println("pulga");
                } else {
                    System.out.println("lagarta");
                }
            } else {
                if (c3.compareTo("hematofago") == 0) {
                System.out.println("sanguessuga");
                } else {
                    System.out.println("minhoca");
                }
            }
        }
        input.close();
    }
}

File: URI/C/Animal.c
#include <stdio.h>

void main()
{
    char c1[15], c2[15], c3[15];

    scanf("%[^\n]s", c1);
    getchar();
    scanf("%[^\n]s", c2);
    getchar();
    scanf("%[^\n]s", c3);
    getchar();

    if (c1[0] == 'v')
    {
        if (c2[0] == 'a')
        {
            if (c3[0] == 'c')
            {
                printf("aguia\n");
            }
            else
            {
                printf("pomba\n");
            }
        }
        else
        {
             if (c3[0] == 'o')
            {
                printf("homem\n");
            }
            else
            {
                printf("vaca\n");
            }
        }   
    }
    else
    {
        if (c2[0] == 'i')
        {
            if (c3[2] == 'm')
            {
                printf("pulga\n");
            }
            else
            {
                printf("lagarta\n");
            }
        }
        else
        {
             if (c3[0] == 'h')
            {
                printf("sanguessuga\n");
            }
            else
            {
                printf("minhoca\n");
            }
        }
    }
    
    
    
}
